django_netjsonconfig/controller/generics.py

```diff
@@ -46,6 +47,7 @@ def get(self, request, *args, **kwargs):
         if bad_request:
             return bad_request
         self.update_last_ip(device, request)
+        checksum_generated.send(sender=device.__class__, checksum=device.config.checksum)
```

Comment:
just pass the device instance, if anyone needs to get the checksum they can drill down.
I'd also provide the request, since we have it and can be useful to developers implementing custom logic.